index.bs

```diff
@@ -11984,6 +11993,118 @@ The [=remote end steps=] given |session| and |command parameters| are:
 
 </div>
 
+## The autofill Module ## {#module-autofill}
+
+The <dfn export for=modules>autofill module</dfn> contains functionality for triggering autofill.
+
+### Definition ### {#module-autofill-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+
+AutofillCommand = (
+  autofill.trigger //
+  autofill.setAddress
+)
+</pre>
+
+### Commands ### {#module-autofill-commands}
+
+#### The autofill.trigger Command ####  {#command-autofill-trigger}
+
+The <dfn export for=commands>autofill.trigger</dfn> command triggers autofill
+on a particular form field using the provided fields.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      autofill.trigger = (
+        method: "autofill.trigger",
+        params: autofill.TriggerParameters
+      )
+
+      autofill.TriggerParameters = {
+        context: browsingContext.BrowsingContext,
+        element: script.SharedReference,
+        field: autofill.Field
+      }
+
+      autofill.Field = {
+        name: autofill.FieldName,
+        value: text
+      }
+
+      autofill.setAddress = (
+        method: "autofill.setAddress",
```

Comment:
I'm a bit confused with this API:
1. `autofill.trigger` has a single param `element` with `field {name, value}`. Does it mean it triggers the autofill only on a single field? What about multi-field addresses?
2. What is the point of `autofill.setAddress`, if the only way to trigger that autofill is `"autofill.trigger`, which provides `values`?

Reply:
We've solved this by moving away from the previous shape into a single `Trigger` call that accepts a card or an address.